features/api_enable.feature

```diff
@@ -0,0 +1,188 @@
+Feature: u.pro.services.enable
+
+    Scenario Outline: u.pro.services.enable.v1 container services
+        Given a `<release>` `<machine_type>` machine with ubuntu-advantage-tools installed
+        When I run `apt-get update` with sudo
+        And I apt install `jq`
+        And I attach `contract_token` with sudo and options `--no-auto-enable`
+        
+        # Basic enable
+        And I run shell command `pro api u.pro.services.enable.v1 --args service=esm-infra | jq .data.attributes` with sudo
+        Then I will see the following on stdout:
+        """
+        {
+          "disabled": [],
+          "enabled": [
+            "esm-infra"
+          ],
+          "messages": [],
+          "reboot_required": false
+        }
+        """
+        When I run shell command `pro api u.pro.status.enabled_services.v1 | jq .data.attributes.enabled_services ` with sudo
```

Comment:
We recently landed a PR which brings simpler steps, like `verify that esm-infra is enabled` - we could use that here.